services/brig/test/resources/elasticsearch-credentials.yaml

```diff
@@ -0,0 +1,2 @@
+username: "elastic"
+password: "changeme"
```

Comment:
It feels wrong to have this file because for RabbitMQ we have environment variables. We should be consistent.

Reply:
Discussed in chat. there are other secrets which are configured in files. So we're already not consistent.
To avoid spending more time on this, we can make sure the helm charts are configured in a way that ensures we can change our minds on this later.